src/wasi.rs

```diff
@@ -366,10 +366,14 @@ pub const _SC_PAGE_SIZE: ::c_int = _SC_PAGESIZE;
 pub const _SC_IOV_MAX: c_int = 60;
 pub const _SC_SYMLOOP_MAX: c_int = 173;
 
+#[allow(unused_unsafe)] // `addr_of!(EXTERN_STATIC)` is now considered safe
```

Comment:
Please add comments on things like this saying something like "we can remove the unsafe when our MSRV is 1.82 or newer", so that they're easy to find when updating MSRV